tests/unit/components/menu/ModelizerSettingsMenu.spec.js

```diff
@@ -72,8 +75,8 @@ describe('Test component: ModelizerSettingsMenu', () => {
     });
 
     describe('Test ref: user', () => {
-      it('should match an empty object', () => {
-        expect(wrapper.vm.user).toEqual({});
+      it('should match a pinia object', () => {
+        expect(wrapper.vm.userStore).not.toEqual({});
```

Comment:
Did not find another way because the object is really big. So my thought was to just check that it was not an empty object !
But I will still add a comment to clarify the test